src/main/java/com/hp/application/automation/tools/run/RunFromFileBuilder.java

```diff
@@ -497,11 +504,10 @@ private String createMtbxFileInWs(FilePath workspace, String mtbxContent) {
 			InputStream in = IOUtils.toInputStream(mtbxContentUpdated, "UTF-8");
 			remoteFile.copyFrom(in);
 
-			String filePath = remoteFile.getRemote();
-			return filePath;
+			return remoteFile.getRemote();
 
 		} catch (IOException | InterruptedException e) {
-			throw new RuntimeException("Failed to save MTBX file : " + e.getMessage());
+			throw new RuntimeException("Failed to save MTBX file : " + e.getMessage(), e);
```

Comment:
Better use checked exception, log and fail the build properly then crash it with RuntimeException.
@m-seldin @radislavB WDYT?